src/Firebase.Auth.Tests/IntegrationTests.cs

```diff
@@ -7,6 +7,7 @@
     using Microsoft.VisualStudio.TestTools.UnitTesting;
     using System.Linq;
 
+    [TestClass]
```

Comment:
Can you remove this attribute? The compiler will detect it and pass it to Test Runner, after which the tests will all fail (missing proper API key) which will fail the entire build